CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing
+
+Welcome to `social-plan-it`! These are some quick notes on how to start contributing to our open source project. A good place to start is to join our [Discord server](https://discord.gg/tTD7PvwpuX).
```

Comment:
Perhaps we can also mention the Monday meetup here as well!

Reply:
Great idea!